packages/grid/x-data-grid-premium/src/hooks/features/rowGrouping/createGroupingColDef.tsx

```diff
@@ -108,19 +96,7 @@ const getGroupingCriteriaProperties = (groupedByColDef: GridColDef, applyHeaderN
 
       return groupingFieldIndexComparator(v1, v2, cellParams1, cellParams2);
     },
-    filterOperators: groupedByColDef.filterOperators?.map((operator) => ({
-      ...operator,
-      getApplyFilterFn: (filterItem, column) => {
-        const originalFn = operator.getApplyFilterFn(filterItem, column);
-        if (!originalFn) {
-          return null;
-        }
-
-        return (params) => {
-          return originalFn(params);
-        };
-      },
-    })),
```

Comment:
Am I mistaken in thinking these ones add nothing?

Reply:
I think you are right 👍